sentiment_lexicon.py

```diff
@@ -1,8 +1,7 @@
-sentiment_lexicon =
-  {
-    "positive": [("totally": .5), ("good": .5), ("smart": .5)],
-    "negative": [("divided": .5), ("not": 1, ("tough": .5)],
-    "neutral": [("country": .5), ("watching": .5), ("looking": .5)]
+sentiment_lexicon = {
+  "positive": [('totally': .5), ('good': .5), ('smart': .5)],
+  "negative": [('divided': .5), ('not': 1, ('tough': .5)],
+  "neutral": [('country': .5), ('watching': .5), ('looking': .5)]
```

Comment:
Inside the tuples go ahead and make the separators commas instead of colons. Tuples use commas to separate items.